src/Workspaces/CSharp/Portable/Rename/LocalConflictVisitor.cs

```diff
@@ -38,11 +38,16 @@ public override void VisitMethodDeclaration(MethodDeclarationSyntax node)
 
         public override void VisitBlock(BlockSyntax node)
         {
+            VisitBlockStatements(node, node.Statements);
```

Comment:
@sys27 My comment was just a comment. I wouldn't consider it actionable unless someone with more knowledge of this component interprets my analysis as an indication that some change is needed.